### ISSUE TITLE
Reading materials from XML file

### ISSUE THREAD

**Issue by lobis** [mipt-npm/gdml.kt#25]
It would be very useful to define materials via an xml file (such as https://github.com/rest-for-physics/materials/blob/2e1d7b0246deecdff70e3e51432e9c55f8862b99/NIST.xml).

Conceptually, one could obtain the materials like this:

```
materials = LoadMaterialsFromXML(xmlfileURI) // perhaps even load from http not just file

materialLead = materials["G4_Pb"] // G4_Pb is a material defined in the file
```

**Comment by altavir** [comment]
Should very easy to do on JVM backend. Thanks for the issue. I will include it in the next version.

**Comment by altavir** [comment]
Implemented in `0.4.0-dev-1`. See [tests](https://github.com/mipt-npm/gdml.kt/blob/master/src/jvmTest/kotlin/space/kscience/gdml/UrlTest.kt) for details. Currently loading both gdmls themselves and materials both from file and from the network.

**Comment by lobis** [comment]
This looks really good! Thanks!
